MaxText/configs/base.yml

```diff
@@ -390,3 +390,6 @@ enable_single_controller: False
 
 # Split physical axes for https://jax.readthedocs.io/en/latest/_autosummary/jax.experimental.mesh_utils.create_device_mesh.html
 allow_split_physical_axes: False
+
+# Attributes needed for the MaxText checkpointing loop using standalone_checkpointer.
+gcs_metrics_bucket: distributed-checkpointing-metrics
```

Comment:
Should there be / is there any ability to provide a folder or path instead of just the bucket?